styles/styles.css

```diff
@@ -1,4 +1,212 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap");
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
did you really want to do margin and padding 0 for everything by default or only the body?